### PR TITLE
PD-1621 / 25.04 / PD-1621 New Emby Tutorial and Screenshots

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR creates a new tutorial for the Emby app based on the new template for apps tutorials.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by micjohnson777** [comment]
Making this a draft until I can confirm the Labels Configuration purpose with Stavros.

File: content/TruenasApps/StableApps/EmbyApp.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+The Emby service uses a Web UI to manage user profiles that are allowed to access your media files and allows you to manage the media library of files.
+Emby naming conventions for media files are important. Pay close attention to the instructions Emby provides to correctly configure the TrueNAS Emby app storage volumes, and when adding any labels to apply to media files.
+
+{{< include file="/static/includes/AppsUnversioned.md" >}}
+
+## Before You Begin
+Before you install the Emby app:
+
+* Read the [Emby Support documentation](https://emby.media/support/articles/Home.html).
+  Emby instructions in the **Media Preparation** and [**Library Setup**](https://emby.media/support/articles/Library-Setup.html) sections, influence how you set up storage volume types and labels used in the TrueNAS **Install Emby** app wizard.
```
Comment (linzi-ix) [inline content/TruenasApps/StableApps/EmbyApp.md:27]:
```suggestion
  Emby instructions in the **Media Preparation** and [**Library Setup**](https://emby.media/support/articles/Library-Setup.html) sections influence how you set up storage volume types and labels used in the TrueNAS **Install Emby** app wizard.
```

File: content/TruenasApps/StableApps/EmbyApp.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+  Organize media files into folders by type, movies, photos, music, etc.
+  Emby instructions say to include the release date of movies in the folder name as follows, *movieName (year)*.
+
+{{< include file="/static/includes/apps/BeforeYouBeginStableApps.md" >}}
+{{< include file="/static/includes/apps/BeforeYouBeginRunAsUser.md" >}}
+
+<div style="margin-left: 33px">{{< trueimage src="/images/SCALE/Apps/EmbyDetailsScreen.png" alt="Nextcloud App Details Screen" id="Nextcloud App Details Screen" >}}</div>
+
+{{< include file="/static/includes/apps/BeforeYouBeginAddNewAppUser.md" >}}
+<div style="margin-left: 33px">The Emby run-as user ID is **0**. The app also includes the default **apps** user ID **568**.
```
Comment (linzi-ix) [inline content/TruenasApps/StableApps/EmbyApp.md:37]: These asterisks aren't rendering as bolded in my build, just plain asterisks.

File: content/TruenasApps/StableApps/EmbyApp.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+* **SMB/CIFS Share (Mounts a volume to a SMB share)** shows settings to create an SMB share storage volume.
+  You must create the SMB share user and share dataset before adding this type.
+* **Anonymous (Temporary directory created on the disk)** creates a temporary directory in the hidden **ix-apps** dataset.
+* **Tmpfs (Temporary directory created on the RAM)** creates a temporary directory in RAM.
+{{< /expand >}}
+
+If adding an SMB share as an additional storage volume, create the SMB dataset and share user(s), and add the user ID for the share user(s) to the dataset ACL.
+{{< include file="/static/includes/apps/InstallWizardStorageSMBOption.md" >}}
+
+### Labels Configuration
```
Comment (linzi-ix) [inline content/TruenasApps/StableApps/EmbyApp.md:176]: Typo in the snippet right about here, says "a present" should be "at present"

File: content/TruenasApps/StableApps/EmbyApp.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+Emby uses metadata to add information to media files, such as where media files are saved, the language used, and parental controls.
+Emby uses tags to add identification information to media files.
+
+{{< include file="/static/includes/apps/InstallWizardLabelsConfiguration.md" >}}
+
+{{< trueimage src="/images/SCALE/Apps/InstallEmbyLabelsConfig.png" alt="Labels Configuration Settings" id="Labels Configuration Settings" >}}
```
Comment (linzi-ix) [inline content/TruenasApps/StableApps/EmbyApp.md:182]: You use the same image for labels as you do for resources

**Review by linzi-ix — CHANGES_REQUESTED** [review]
a few comments

File: content/TruenasApps/StableApps/EmbyApp.md (hunk 3 above)
Comment (micjohnson777) [reply content/TruenasApps/StableApps/EmbyApp.md:176]: Not found.

File: content/TruenasApps/StableApps/EmbyApp.md (hunk 4 above)
Comment (micjohnson777) [reply content/TruenasApps/StableApps/EmbyApp.md:182]: fixed

File: content/TruenasApps/StableApps/EmbyApp.md (hunk 2 above)
Comment (micjohnson777) [reply content/TruenasApps/StableApps/EmbyApp.md]: replaced with html tags.

**Review by linzi-ix — APPROVED** [review]

**Pull request merged**